doc/releases/changelog-0.23.0.md

```diff
@@ -212,167 +326,36 @@
-    [KaHyPar](https://pypi.org/project/kahypar/) which needs to be installed separately.
-    To integrate with the existing low-level manual cut pipeline, method `qcut.find_and_place_cuts()`,
-    which uses `qcut.kahypar_cut()` as the default auto cutter, has been implemented.
-    The automatic cutting feature is further integrated into the high-level interfaces
-    `qcut.cut_circuit()` and `qcut.cut_circuit_mc()` for automatic execution of arbitrary
-    circuits on smaller devices.
-    [(#2330)](https://github.com/PennyLaneAI/pennylane/pull/2330)
-    [(#2428)](https://github.com/PennyLaneAI/pennylane/pull/2428)
-
-<h3>Improvements</h3>
-
-* Added the `QuantumTape.shape` method and `QuantumTape.numeric_type`
-  attribute to allow extracting information about the shape and numeric type of
-  quantum tapes.
-  [(#2044)](https://github.com/PennyLaneAI/pennylane/pull/2044)
-
-* Defined a `MeasurementProcess.shape` method and a
-  `MeasurementProcess.numeric_type` attribute.
-  [(#2044)](https://github.com/PennyLaneAI/pennylane/pull/2044)
-
-* The parameter-shift Hessian can now be computed for arbitrary
-  operations that support the general parameter-shift rule for
-  gradients, using `qml.gradients.param_shift_hessian`
-  [(#2319)](https://github.com/XanaduAI/pennylane/pull/2319)
-
-  Multiple ways to obtain the
-  gradient recipe are supported, in the following order of preference:
-
-  - A custom `grad_recipe`. It is iterated to obtain the shift rule for
-    the second-order derivatives in the diagonal entries of the Hessian.
+<h4>New transform for transpilation ⚙️ </h4>
 
-  - Custom `parameter_frequencies`. The second-order shift rule can
-    directly be computed using them.
+* Added a swap based transpiler transform.
+  [(#2118)](https://github.com/PennyLaneAI/pennylane/pull/2118)
 
-  - An operation's `generator`. Its eigenvalues will be used to obtain
-    `parameter_frequencies`, if they are not given explicitly for an operation.
+  The transpile function takes a quantum function and a coupling map as inputs
+  and compiles the circuit to ensure that it can be executed on corresponding
+  hardware. The transform can be used as a decorator in the following way:
 
-* Most compilation transforms, and relevant subroutines, have been updated to
-  support just-in-time compilation with `jax.jit`.
-  [(#1894)](https://github.com/PennyLaneAI/pennylane/pull/1894/)
+  ```python
+  dev = qml.device('default.qubit', wires=4)
 
-* The `qml.specs` transform now accepts an `expansion_strategy` keyword argument.
-  [(#2395)](https://github.com/PennyLaneAI/pennylane/pull/2395)
+  @qml.qnode(dev)
+  @qml.transforms.transpile(coupling_map=[(0, 1), (1, 2), (2, 3)])
+  def circuit(param):
+      qml.CNOT(wires=[0, 1])
+      qml.CNOT(wires=[0, 2])
+      qml.CNOT(wires=[0, 3])
+      qml.PhaseShift(param, wires=0)
+      return qml.probs(wires=[0, 1, 2, 3])
+  ```
```

Comment:
can we include output here? For example, might be good to include a circuit drawing showing the compiled circuit